src/Base/Repositories/AbstractRepository.php

```diff
@@ -235,7 +235,9 @@ public function transform(WP_Post $post)
             'post_status' => $post->post_status
         ];
 
-        $data = $this->assignFields($data, $post);
+		$data = apply_filters( 'ok_geo_rest_add_geodata', $data, $post );
```

Comment:
Mijn voorstel zou zijn om de naamgeving van deze filter generieker te maken: `owc/openpub/base/rest-add-fields`

Reply:
@mvdhoek1 : Wij willen dat hij toegevoegd kan worden in verschillende plugins van andere leveranciers. Jullie kunnen hem in de OpenPub toevoegen om aan de OpenPub de geodata toe te voegen aan de REST API en een andere partij kan dezelfde code toevoegen aan hun plugin om de geodata toe te voegen aan hun plugin. Om die reden willen we geen naamgeving gebruiken waar openpub in zit. Wel kunnen we hem `owc/openkaarten/geodata/rest-add-fields` of iets dergelijks noemen. Hoe lijkt dat je?

Reply:
Oh op die manier, dan begrijp ik het. Dan is `owc/openkaarten/geodata/rest-add-fields` wmb akkoord!

Reply:
@mvdhoek1 : Ik heb de naam van het filter aangepast. Als jullie het hebben doorgevoerd en aan onze kant is de Geodata plugin te gebruiken, dan kunnen we testen of het werkt op deze manier, wat we voor ogen hebben.
Dank alvast!